docs/html-css/week-1/lesson.md

```diff
@@ -44,13 +63,41 @@ All together, let's review the basic syntax in the following example:
 </article>
 ```
 
-Which parts are the **Tags** and which parts are the **Attributes**.
+Which parts are the **tags** and which parts are the **attributes**?
+
+## HTML structure
```

Comment:
The base structure of an HTML document is added so students get some basic understanding of what these tags are that they see in every HTML document they work with.